index.html

```diff
@@ -711,12 +712,12 @@ <h3>
         </p>
         <p>
           Certain UI/UX concerns and/or platform conventions will mean that
-          some screen orientations and display modes <dfn>cannot be used
-          together</dfn>. Which orientations and display modes cannot be used
-          together is left to the discretion of implementers. For example, for
-          some user agents, it might not make sense to change the <a>default
-          screen orientation</a> of an application while in `browser`
-          <a>display mode</a>.
+          some screen orientations and <dfn>cannot be used together</dfn>.
+          Which orientations and display modes cannot be used together is left
+          to the discretion of implementers. For example, for some user agents,
```

Comment:
should "user agent" link to something?

Reply:
good idea... to infra, actually 😂

Reply:
I might do that as a followup, as there are like 100+ instances of "user agent".